spec/fixtures/application/1.0/maat_application.json

```diff
@@ -4,6 +4,7 @@
   "reference": 6000001,
   "application_type": "initial",
   "submitted_at": "2022-10-24T09:50:04.019Z",
+  "signed_at": "2022-10-24T09:50:04.000+00:00",
```

Comment:
`submitted_at` and `signed_at` will be identical with our current implementation, so probably want to make that also explicit by using same timestamp format including same milliseconds?

Reply:
🤦‍♀️ I had an old version of the file before I updated my branch so copied over the wrong value, will change